package.json

```diff
@@ -16,86 +16,95 @@
     "url": "https://github.com/adobe/aem-react-editable-components/issues"
```

Comment:
I guess with this change we need to change a major version of this repo.

Reply:
Indeed, this will be v2 as there are quite a few breaking changes